openbook_communities/views/communities/serializers.py

```diff
@@ -30,7 +30,7 @@ class CreateCommunitySerializer(serializers.Serializer):
                                           max_upload_size=settings.COMMUNITY_AVATAR_MAX_SIZE)
     cover = RestrictedImageFileSizeField(required=False,
                                          max_upload_size=settings.COMMUNITY_COVER_MAX_SIZE)
-    invites_enabled = serializers.BooleanField(required=False, allow_null=False)
+    invites_enabled = serializers.NullBooleanField(required=False)
```

Comment:
Nope, use `NullBooleanField` only when you want `allow_null=True` .. by default null will be false on a `BooleanField`.